### PR TITLE
Updates & Fixes

### PR DESCRIPTION
**Pull request by dennisarimi**
- Changed images used from svg to png/jpg/jpeg
- Corrected text errors
- Updated results/resources content; added new file, changed file/link dates, changed titles and descriptions
- Updated team images, order and corrected description text errors
- Results page name change to Impact

### CODE REVIEW

File: src/components/Timeline/timeline.tsx [hunk 1]
```diff
@@ -18,43 +18,87 @@ const Timeline = (props: IProps) => {
...
 					<div className={styles.card}>
-						<h4>Preparation and Data Collection</h4>
+						<h4>Capacity Building and Data Collection</h4>
 						<p>
 							Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec volutpat ac lacus ut ultricies. Proin
 							cursus
 						</p>
+						{/* <ul>
+							<li>Women RISE launch meeting in Nairobi, Kenya and forming the project team.</li>
+							<li>Ongoing monthly team meetings.</li>
```
Comment (MatthewOlaka) [inline src/components/Timeline/timeline.tsx:30]: Any reason why we have these descriptions added but commented out??

File: src/components/Timeline/timeline.tsx [hunk 2]
```diff
@@ -18,43 +18,87 @@ const Timeline = (props: IProps) => {
...
 					<div className={styles.card}>
-						<h4>Co-creations and Inventions</h4>
+						<h4>Data Analysis and Co-Created Interventions</h4>
 						<p>
 							Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec volutpat ac lacus ut ultricies. Proin
 							cursus
 						</p>
+						{/* <ul>
+							<li>12 FGDs for stakeholder mapping of the agricultural sector.</li>
+							<li>3 county-level human-centred design health ideation workshops involving coop officers from 9 agricultural cooperatives engaging 50 participants.</li>
```
Comment (MatthewOlaka) [inline src/components/Timeline/timeline.tsx:54]: Same as above

File: src/components/Timeline/timeline.tsx [hunk 3]
```diff
@@ -18,43 +18,87 @@ const Timeline = (props: IProps) => {
...
 					<div className={styles.card}>
-						<h4>Project Monitoring</h4>
+						<h4>Gender-Transformative Training and Project Monitoring</h4>
 						<p>
 							Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec volutpat ac lacus ut ultricies. Proin
 							cursus
 						</p>
+						{/* <ul>
+							<li>Household and cooperative gender-transformative training manual developed centred around the “Family Life Model.”</li>
+							<li>54 household-level and 18 cooperative-level two-day gender-transformative workshops.</li>
```
Comment (MatthewOlaka) [inline src/components/Timeline/timeline.tsx:75]: Same as above

File: src/components/Timeline/timeline.tsx [hunk 4]
```diff
@@ -18,43 +18,87 @@ const Timeline = (props: IProps) => {
...
+						</ul> */}
 					</div>
 				</div>
+
+				{/* <div className={`${styles.item} ${styles.item_4}`}>
+					<span className={styles.icon}>
+						<FontAwesomeIcon icon={''} />
+					</span>
+					<span className={styles.date}>Apr 2024 - Sep 2024</span>
+					<div className={styles.card}>
```
Comment (MatthewOlaka) [inline src/components/Timeline/timeline.tsx:87]: Same as above

**Review by MatthewOlaka — COMMENTED** [review]
Looks good! just confused on the commented out code...

File: src/components/Timeline/timeline.tsx (hunk 1 above)
Comment (dennisarimi) [reply src/components/Timeline/timeline.tsx:30]: For Sigi when he comes to work on the CSS, making lives easier😘

Comment (MatthewOlaka) [reply src/components/Timeline/timeline.tsx:30]: 🥹

Comment (MatthewOlaka) [reply src/components/Timeline/timeline.tsx:30]: when he looks after you 💦

**Review by MatthewOlaka — APPROVED** [review]
🧶

File: src/components/Timeline/timeline.tsx (hunk 1 above)
Comment (dennisarimi) [reply src/components/Timeline/timeline.tsx:30]: just approve my ting😂 that's me and Sigi's secret😉

**Pull request merged**